prow/config/jobs/istio.yaml

```
@@ -76,6 +76,20 @@ jobs:
       - name: INTEGRATION_TEST_FLAGS
         value: --istio.test.skipVM
 
+  - name: integ-ambient-calico
+    requirements: [kind]
```

```suggestion
  - name: integ-ambient-calico
    modifiers: [presubmit_optional]
    requirements: [kind]
```

I think we should make it optional until we know it's working as expected